Say Their Names/Source/Controller/Petitions/PetitionsViewCode.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  PetitionsUI.swift
+//  Say Their Names
+//
+//  Created by Kyle Lee on 6/2/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+/// The UI for Petitions
+final class PetitionsViewCode: UIView {
+    
+    private lazy var petitionsLabel: UILabel = {
```

Comment:
```suggestion
    private let petitionsLabel: UILabel = {
```
will work the same, just without the ability to accidentally overwrite the variable 🙂